14_functions_lab/04.py

```diff
@@ -1,6 +1,20 @@
-"""
-Write a function that takes minlen and
-a list of words, and returns only the words
-longer than minlen
-"""
-
+"""
+function return words greather than length
+"""
+from types import *
+
+
+
+def func(length,*args):
+    res = []

```

Comment:
Note how in first glance it's not that obvious what the code returns. Using list comprehension I think you can re-order the code to make it clearer